### ISSUE TITLE
Should validation work in watch mode?

### ISSUE THREAD

**Issue by Ben-at-h** [TypeStrong/typedoc#2584]
## Search terms

validation, notDocumented, requiredToBeDocumented, watch mode.

## Question
Should warnings show up when using TypeDoc in watch mode? 
I have been running TypeDoc with "notDocumented" set to true and the standard "requiredToBeDocumented" options as suggested in the online documentation. When I run TypeDoc out of watch mode I get around 20 warnings as expected, but when using watch mode on the same code I get the following:
```
[info] Starting compilation in watch mode...
[info] Found 0 errors. Watching for file changes.
[info] Documentation generated at ./docs
```

**Comment by Gerrit0** [comment]
I'm pretty sure that's just an oversight, should be an easy fix!

**Comment by Gerrit0** [comment]
Fixed with 0.26, which is releasing 2024/06/21
